pathfinder/api/rest.py

```diff
@@ -107,6 +108,14 @@ def get(self, token_network_address: str):
 
         return {'result': paths}, 200
 
+class InfoResource(PathfinderResource):
+    def get(self):
+        ip = socket.gethostbyname(socket.gethostname()) ## this is the internal IP address, which is a feature that will be scoped out soon anyway
+        settings = 'PLACEHOLDER'
+        version = '0.1'
+        dude_who_runs_it = 'Dominik'
+
+        return {'IP': ip, 'Settings': settings, 'Version': version, 'Dude who runs it': dude_who_runs_it}, 200
```

Comment:
Let's see what stuff we need here, as you wrote IP doesn't make to much sense.

`Dude who runs it` should probably be `operator`.

You should be able to get the version from the package info, see how raiden does it: https://github.com/raiden-network/raiden/blob/45c4c47ec08e1e232b5f0b24386c8fc0a46f647a/raiden/utils/__init__.py#L173

And for me, maybe a `message` field with some info that would be displayed in the Raiden client might make sense as well.